packages/functions/src/schema/index.ts

```diff
@@ -25,11 +25,13 @@ export default createRulesetFunction<unknown, Options>(
         },
         dialect: {
           enum: ['auto', 'draft4', 'draft6', 'draft7', 'draft2019-09', 'draft2020-12'],
+          default: 'auto',
```

Comment:
unrelated, but I thought it'd be actually cool to express what the default is.
Going to be useful later on in Studio when building forms.